Makefile

```diff
@@ -100,3 +100,43 @@ docker-shell:  ## SSH into Docker container
 .PHONY: docker-destroy
 docker-destroy:  ## Tear down the Docker containers
 	docker compose down --volumes
+
+.PHONY: docker-dev-pull
+docker-dev-pull: load-design-system-templates  ## Pull dev Docker containers
+	docker compose -f docker-compose-dev.yml pull
+
+.PHONY: docker-dev-up
+docker-dev-up:  ## Start dev Docker containers
+	docker compose -f docker-compose-dev.yml up --detach 
+
+.PHONY: docker-dev-stop
+docker-dev-stop: ## Stop dev Docker containers
+	docker compose -f docker-compose-dev.yml stop
+
+.PHONY: docker-dev-down
+docker-dev-down: ## Stop dev Docker containers
+	docker compose -f docker-compose-dev.yml down
+
+.PHONY: check-makemigrations
+check-makemigrations: ## Convenience alias to run makemigrations in venv
+	poetry run python ./manage.py makemigrations --check
+
+.PHONY: makemigrations
+makemigrations: ## Convenience alias to run makemigrations in venv
+	poetry run python ./manage.py makemigrations
+
+.PHONY: collectstatic
+collectstatic:  ## Convenience alias to run collectstatic in venv
```

Comment:
We should replace these with comments that describe what the command does. All make targets are convenience aliases so this doesn't add much value.